engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -527,6 +567,33 @@ public JSONObject getLogicalOperator(@PathParam("operatorName") String operatorN
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName:.+}")
```

Comment:
:.+ was required if separator is '/', as internal module names and operators populated by the module will have '/' in their names.
Current REST api don't handle '/' in names of any component correctly. I have opened following Jira for it. Will handle this thing from that PR, I will remove :.+ for now.
https://issues.apache.org/jira/browse/APEXCORE-282
do you see other problem with component having '/' in their names, If thats the case then I will change the separator. The top voted separator was '/' and next is '$'.